### ISSUE TITLE
How to create LMDB version of training data?

### ISSUE THREAD

**Issue by ntvy95** [amandajshao/Slicing-CNN#3]
Thank you very much for your work.

When I try to run with your LMDB version of training data, I ran into this error message:

> F0302 08:00:34.350658  9538 db_lmdb.hpp:13] Check failed: mdb_status == 0 (2 vs. 0) No such file or directory

Though, your LMDB version of test data works just fine. 

I suspect this may be due to the loss of data during transmission, I tried to re-download the LMDB version of the training data multiple times but none of them works.

I am thinking of creating LMDB version of training data, however, I am not sure how to do this. I looked into your LMDB version of test data and it looks like you only take the first 100 even frames of the video. However, I am not sure about width and height of the encoded frame data as their length vary (which is `len(np.fromstring(datum.data, dtype=np.uint8))`, in the full code:

```
import caffe
import lmdb
import numpy as np
import matplotlib.pyplot as plt
from caffe.proto import caffe_pb2

lmdb_file = "crowd_data_lmdb_test"
lmdb_env = lmdb.open(lmdb_file)
lmdb_txn = lmdb_env.begin()
lmdb_cursor = lmdb_txn.cursor()
datum = caffe_pb2.Datum()

for key, value in lmdb_cursor:
    datum.ParseFromString(value)
    print len(np.fromstring(datum.data, dtype=np.uint8))
```
). Further more, I have no information in regards to `datum.channels`, `datum.height `and `datum.width` which is required by `caffe.io.datum_to_array`.

May I ask what is the height and width of each frame stored in LMDB?

Thank you very much for your time and consideration.

**Comment by ntvy95** [comment]
I have realized I was overthinking about it. The file data.mdb is missed in the folder crowd_data_lmdb_train_label. Here I upload the missing data.mdb for convenience: https://drive.google.com/file/d/1aLuDng-V-SEHONTPRzrZ84sh2lz1yNpr/view?usp=sharing
